packages/terra-worklist-data-grid/src/subcomponents/ColumnHeader.jsx

```diff
@@ -21,7 +21,7 @@ const propTypes = {
    */
   tabStopColumnIndex: PropTypes.number,
   /**
-   * Function that is called when a selectable header cell is selected. Parameters: `onColumnSelect(columnId)`.
+   * Function that is called when a selectable header cell is selected. Parameters: `function(columnId)`.
```

Comment:
Would it be ok to also add
`@param {string} columnId The selected column header`
or something similar?

Reply:
Good idea!
Added here: 69e4d8c

The definitions of each prop can be done in the documentation Jira.